frontend/src/modules/controllers/RotaryEncodeController.tsx

```diff
@@ -41,31 +45,39 @@ export const RotaryEncodeController = ({
   max = 100,
   isActive,
 }: Props): JSX.Element => {
-  const rotaryEncoder = useSelector(getRotaryEncoder, shallowEqual);
+  const step = useSelector(getRotaryEncoderStep, shallowEqual);
+  const stepDiff = useSelector(getRotaryEncoderStepDiff, shallowEqual);
```

Comment:
Can we get rid of the `shallowEqual` in these `useSelector`s, now that we're using selectors which return numbers rather than an object?